refgenconf/const.py

```diff
@@ -9,7 +9,7 @@
 CFG_NAME = "genome configuration"
 CFG_ENV_VARS = ["REFGENIE"]
 CFG_CONST = ["CFG_NAME", "CFG_ENV_VARS"]
-DEFAULT_SERVER = "http://refgenomes.databio.org"
+DEFAULT_SERVER = ["http://refgenomes.databio.org"]
```

Comment:
It's slightly confusing to bind a collection to a name that's a singular noun. It may not be worth the cost of renaming, but I think it's worth considering the tradeoff between the short term renaming cost and the longer term potential confusion. No strong opinion, though.

Reply:
Are you suggesting to change the name of the constant variable here (based on where you've placed this comment)? Or you also mean changing the `genome_server` config key to `genome_servers`? which is actually a good point

Reply:
I think he meant the const variable here...

but yeah it brings up the config thing. 

I'd say for the const thing let's just move it back to singular... we don't really need multiple defaults do we?

Reply:
actually, whatever. just make them all plural if you want. I'm fine with whatever.

Reply:
yeah, we don't need multiple default servers, so I'll stick to singular noun and make it a string.

As for the server config key, I'll make it `genome_servers` but the change will be backwards compatible (whatever is in `genome_server` is moved over to the `genome_servers` now and `genome_server` is removed). And we'll start advertising the new key from now on.

The question is, does that require config format version bump? I'd prefer not to change it yet.

Reply:
it's fine with me either way.

Reply:
ok. It's ready to release then